data/applications/src/main/java/de/mm20/launcher2/applications/AppRepository.kt

```diff
@@ -222,7 +222,7 @@ internal class AppRepositoryImpl(
                     appResults.addAll(apps)
                 } else {
                     appResults.addAll(apps.filter {
-                        matches(it.label, query)
+                        matches(it.label, query) || matches(it.componentName.packageName.split('.').last(), query)
```

Comment:
why only the last part?

Reply:
Because of all the `com` prefixes and company names, e.g. thus any `com` search queries will break.

I'm thinking about using English (or default) label instead by loading context, what do you think? Although non-English apps may not be indexed, however in that situation it may not be a problem

Reply:
Package names in general are pretty inconsistent (this is the reason I don't use them already). Loading the English label is certainly the better approach, as long as you cache the English label (just add an internal field to `LauncherApp` and store it alongside the app)

Reply:
Sorry for late response, but tinkering around with it it looks like loading `Context` with different locale config doesn't actually load different labels for apps, since it's being handled by `PackageManager.getText` which, as I understand, is manufacturer dependent. I'll see if I can retrieve it from adb/manifest somehow, but it looks like package name may be an option